app/src/actioncommands.cpp

```diff
@@ -757,13 +758,11 @@ void ActionCommands::reverseSelectedFrames()
     }
 
     QList<int> selectedIndexes = currentLayer->selectedKeyFramesPositions();
-    int swapEnd = selectedIndexes.count()-1;
 
-    for (int swapBegin = 0; swapBegin < swapEnd; swapBegin++) {
+    for (int swapBegin = 0, swapEnd = selectedIndexes.count()-1; swapBegin < swapEnd; swapBegin++, swapEnd--) {
```

Comment:
I usually prefer the former solution and find it easier to read but if it makes you happy, then let's roll with your change.

Reply:
The main reason for this change is that I wanted to keep the `swapEnd--` near the rest of the loop instructions. If you find the `swapEnd = …` more readable as a separate instruction I’m fine with that since it’ll still be right next to the loop head, I just didn’t like the `swapEnd--` at the very end of the loop body.